src/System.IO.Packaging/src/System/IO/Packaging/PackUriHelper.cs

```diff
@@ -988,7 +985,8 @@ private static PackUriHelper.ValidatedPartUri GetPartUriComponent(Uri packUri)
         private static readonly string s_relationshipPartExtensionName = ".rels";
 
         // Forward Slash
-        internal static readonly char ForwardSlashChar = '/';
+        internal const char ForwardSlashChar = '/';
+        internal static readonly char[] ForwardSlashSeparator = { '/' };
```

Comment:
This should probably be named ForwardSlashCharArray, or something like that which connects it by name to ForwardSlashChar.  We could also define this in terms of ForwardSlashChar, e.g.
```C#
internal static readonly char[] ForwardSlashCharArray = { ForwardSlashChar };
```